src/databricks/labs/ucx/source_code/known.json

```diff
@@ -237,6 +237,14 @@
     "audioread.rawread": [],
     "audioread.version": []
   },
+  "autocommand": {
```

Comment:
revert changes to this file

Reply:
done- shows this file is not being modified now, sorry abt that!